frontend/src/pages/-/health/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { useLazyQuery, useQuery } from "@apollo/client";
+import { GET_USER_PROFILE } from "@graphql/users/queries";
+import { addApolloState, initializeApollo } from "@lib/apolloClient";
+import { GetServerSideProps, InferGetServerSidePropsType } from "next";
+import { User } from "src/types/users";
+
+const HealthPage = ({ user }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
```

Comment:
If we use the props from getServerSideProps it works out just fine, except any updates to that state through mutations etc. will not trigger a rerender and an updated state. Therefore, we must also make the queries, which will hit the cache, and use that data to have a state that will be updated.